_partials/_migrate_dual_write_step2.md

```diff
@@ -0,0 +1,14 @@
+## 2. Modify the application to write to a secondary database
+
+How exactly to do this is dependent on the language that your application is
+written in, and on how exactly your ingestion and application function. In the
+simplest case, you simply execute two inserts in parallel. In the general case,
+you will need to think about how to handle the failure to write to either the
+old or new database, and what mechanism you want to or can build to recover
+from such a failure.
+
+You may also want to execute the same read queries on the old and new database,
```

Comment:
The point about correctness is mentioned in point 7 "Validate that all data is present in target database", so we can remove this paragraph from here and keep this section specific to "modifying application for dual writes" only. Feel free to ignore if you think so.

Reply:
Ya, Vineeth suggested the same thing. I think that I would prefer to keep this in here, purely from the point of view that people who are in the process of modifying the write path of their application may wonder "what about the read path?".